routers/web/admin/admin.go

```diff
@@ -140,6 +141,13 @@ func Dashboard(ctx *context.Context) {
 	ctx.HTML(http.StatusOK, tplDashboard)
 }
 
+func SystemStatus(ctx *context.Context) {
+	// FIXME: update periodically
```

Comment:
comment obsolete?

Reply:
I don't know, it was there so I left it there

Reply:
Where?
To me, it looks like a new comment…

Reply:
https://github.com/go-gitea/gitea/blob/4feb91f8574e2363d2120655562e6f09bbf1ffcb/routers/web/admin/admin.go#L135

Reply:
Isn't the point of this PR to implement what the comment suggests? 😆

Reply:
I thought it meant "don't update on each HTTP request, cache the status and update the cache every X sends/minutes/hours"

You do bring up an interesting point...

Reply:
Not sure if caching is necessary. How long does the function take on your system?

Reply:
~1-4 ms

Reply:
Yeah, that can be live with no cache.

Reply:
I will remove the comment today

Reply:
https://github.com/go-gitea/gitea/pull/29163/commits/969352d56322eb7fcca06b9f43c0a01ed8da8f4c